### PR TITLE
Amazon-AWS-EC2-Volumes - AGM 12372, 12373, 12374.

### PR DESCRIPTION
**Pull request by mihaitusa**
Create, attach and detach volume service methods.
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com>

### CODE REVIEW

**Comment by mihaitusa** [comment]
@gituser173 Please review it!

File: score-jClouds/src/main/java/io/cloudslang/content/jclouds/utils/InputsUtil.java [hunk 1]
```diff
@@ -34,14 +34,19 @@ private InputsUtil() {
...
-        }
+    public static String getDefaultDelimiter(String delimiter) {
+        return StringUtils.isBlank(delimiter) ? COMMA_DELIMITER : delimiter;
+    }
+
+    public static int getValidInstanceStateCode(String input) {
+        return InstanceState.getKey(input);
+    }
+
+    public static boolean getBoolean(String input) {
```
Comment (gituser173) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/utils/InputsUtil.java]: rename this method if you need true as default

Comment (mihaitusa) [inline score-jClouds/src/main/java/io/cloudslang/content/jclouds/utils/InputsUtil.java]: @gituser173 sure....

**Comment by gituser173** [comment]
:+1: you can merge it if it's OK from the functional standpoint

**Pull request merged**